test/hotspot/jtreg/compiler/c2/irTests/scalarReplacement/AllocationMergesTests.java

```diff
@@ -197,6 +213,23 @@ public void runner(RunInfo info) {
             }
         }
 
+       try {
```

Comment:
Do we have tests with multiple nested Phis? Including with nested objects fields?

Would you be so kind to add a comment at the top of this class or method describing all scenarios that are tested?